### PR TITLE
Added the APIs using swagger

### PR DESCRIPTION
**Pull request by ansjin**
Added the swagger framework for adding the APIs.

### CODE REVIEW

**Comment by sacdallago** [comment]
Why is swagger here and not [here](https://github.com/MusicConnectionMachine/api)?

**Comment by sacdallago** [comment]
Also: there are conflicts before this can be merged, similar situation to [this](https://github.com/MusicConnectionMachine/StructuredData/pull/34#issuecomment-287297060)

**Comment by Henni** [comment]
@sacdallago we run our relationship extraction algorithms inside separate containers. These support a simple API to receive sentences and afterwards return the relationships they extracted.

This has nothing to do with the database API and is only relevant for our own project.

**Comment by sacdallago** [comment]
@Henni thanks for the clarification :)

**Review by krishenk — COMMENTED** [review]
Looks good, but make sure that you align the Dockerfile with the updated version of node, and clean up the extra empty lines.

File: algorithms/ollie/api/swagger/swagger.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ paths:
         \ in the text by running ollie algorithm on it. The response includes the\
         \ sentence and the instances which contains detail about each instance of\
         \ relationship found. \n"
-      operationId: "getRelationshipsGET"
+      operationId: "getRelationshipsPOST"
```
Comment (Henni) [inline algorithms/ollie/api/swagger/swagger.yaml]: This is nitpicking, but as this isn't a real REST call and more an RPC I suggest to not use the REST terminology and name this `extractRelationships`.

**Review by Henni — CHANGES_REQUESTED** [review]
Except for the single comment looks good (even though I haven't worked with swagger yet)

File: algorithms/ollie/api/swagger/swagger.yaml (hunk 1 above)
Comment (ansjin) [reply algorithms/ollie/api/swagger/swagger.yaml]: yeah True, but actually swagger automatically add the GET and POST to the operations. But for our purpose I have renamed them

**Review by Henni — APPROVED** [review]

**Pull request merged**